lib/openskill.ex

```diff
@@ -145,4 +145,39 @@ defmodule Openskill do
       result
     end
   end
+
+  @spec predict_win([[{float(), float()}]]) :: [float()]
+  def predict_win(teams) do
```

Comment:
It might be nice to include the comment
```ex
  @doc """
  Predict the win probability for each team.

  ## Examples

      iex> teams = [
      ...>   [{25, 8.333}, {30, 6.666}],
      ...>   [{27, 7.0}, {28, 5.5}]
      ...> ]
      iex> Openskill.predict_win(teams)
      [0.7310585786300049, 0.2689414213699951]

  """
  ```
and use one of the existing types `mu_sigma_pair()`  for documentation and consistency with other functions.

It would also make it possible to potentially open a PR to original repo because the changes so far are not BAR specific.